### PR TITLE
Introduction to Ensemble Methods in Machine Learning

### LINKED ISSUE #5431: Introduction to Ensemble Methods in Machine Learning
### Topic Suggestion
Introduction to Ensemble Methods in Machine Learning

#### Proposed title of article
Introduction to Ensemble Methods in Machine Learning
#### Proposed article introduction
In Machine Learning, the ensemble method is defined as a multimodal system in which different classifiers and techniques are strategically combined into a predictive model (grouped as Sequential Model, Parallel Model, Homogeneous and Heterogeneous methods etc.) 
The ensemble method also aids in reducing variance in predicted data, minimizing bias in the predictive model, and accurately classifying and predicting statistics from complex problems.

#### Key takeaways 
Types of Ensemble Methods in Machine Learning
Technical Classification of Ensemble Methods
Heterogeneous Ensemble
 Homogeneous Ensemble

#### Article quality
Since heterogeneous Ensemble and homogeneous Ensemble are the major types of Ensemble methods, in my article I will give the reader a clear understanding about tem and even go a step further to elaborate about technical classification of Ensemble Methods. I'll make sure to use simple grammar hence my article will be beginner friendly.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by EssyH**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #5431 

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by EssyH** [comment]
this closes #5431

**Comment by hectorkambow** [comment]
Good afternoon and thank you for submitting your article for publishing.
Your article has been entered into our queue and should be reviewed (and published pending any changes) as soon as a peer reviewer is finished with the ones in the queue before it. 🚀
Thank you.

**Review by srishilesh — CHANGES_REQUESTED** [review]
@EssyH 

This PR looks incomplete. It doesn't have a proper conclusion.

Moreover, this PR is too short on content. And, it doesn't convey much about the concepts.

Already, we have many article on "Ensemble learning". Have a look at them [here](https://www.section.io/engineering-education/search/?q=ensemble). See how they've formatted, what the flow is, and how you could improve this article.

Markdown formatting doesn't look good. I would suggest you to take look at other articles for improving on it.

Is your PR an article or tech-tutorial (does it contain code snippets)?

Since, this PR is low on content and requires some major revisions on formatting and content, I will close this PR for now. You can feel free to re-open a new PR and ping me there.

If you have any concerns, feel free to reply back.

Thanks.

**Comment by EssyH** [comment]
Hi @srishilesh 
thanks for the info, though I was kindly asking if you could tell me  the specific markdown formatting that i did wrongly.  My article is a tech tutorial. About the conclusion part, I think i forgot mistakenly but I'll work on it

File: content/articles/introduction-to-ensemble-methods-in-machine-learning/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Introduction to Ensemble Methods in Machine Learning
+
+## Prerequisites
```
Comment (srishilesh) [inline content/articles/introduction-to-ensemble-methods-in-machine-learning/index.md:3]: As mentioned in the guidelines (also in the PR checklist), the markdown headers must start from `###`, not `##`

Comment (srishilesh) [reply content/articles/introduction-to-ensemble-methods-in-machine-learning/index.md:3]: Bring pre-requisites section after table of contents

File: content/articles/introduction-to-ensemble-methods-in-machine-learning/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# Introduction to Ensemble Methods in Machine Learning
+
+## Prerequisites
+- The reader should have basic knowledge in machine learning algorithms.
+- The reader have read about machine learning life cycles and its advantages.
+
+
+## Introduction
```
Comment (srishilesh) [inline content/articles/introduction-to-ensemble-methods-in-machine-learning/index.md:8]: `###`

File: content/articles/introduction-to-ensemble-methods-in-machine-learning/index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+### Types of Ensemble Methods in Machine Learning
+In machine learning, bias, diversity, and noise all have a negative impact on errors and predictions in machine learning models. A combination of several methods is used to avoid these issues. They include ;
+   - Homogeneous Ensemble
+   - Heterogeneous Ensemble
+   - Sequential Ensemble Methods
+   - Parallel Method
+
+
+#### ***1. Homogeneous Ensemble***
```
Comment (srishilesh) [inline content/articles/introduction-to-ensemble-methods-in-machine-learning/index.md:41]: You can remove `***` and the numbers as well

File: content/articles/introduction-to-ensemble-methods-in-machine-learning/index.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+In this ensemble method, the base learner is produced in parallel sequence, with no data dependency. Every piece of data in the base learner is obtained on its own.
+
+
+### Technical Classification of Ensemble Methods
+Ensemble methods are technically classified as ;
+
+                      - Boosting
+                      - Bagging
+                      - Random forest
+                      - Stacking
```
Comment (srishilesh) [inline content/articles/introduction-to-ensemble-methods-in-machine-learning/index.md:65]: You need to indent them properly

File: content/articles/introduction-to-ensemble-methods-in-machine-learning/index.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+#### ***(a) Boosting***
+Booosting ensemble is an ensemble method that constructs a strong classifier from a group of weak classifiers by merging multiple weak classifiers in series. It's an example of sequential ensemble methods whose main aim/property is the concept of correcting prediction errors.
+
+>Below is an example of how you implement boosting in python
+
+![Implementing boosting in python](/engineering-education/introduction-to-ensemble-methods-in-machine-learning/boosting.jpg)
+
+#### ***(b) Bagging***
```
Comment (srishilesh) [inline content/articles/introduction-to-ensemble-methods-in-machine-learning/index.md:74]: You can remove `***`

**Review by srishilesh — COMMENTED** [review]
@EssyH 

These are few of the markdown formatting mistakes.

Also, as we have mentioned in the guidelines, a tech-tutorial contains code snippets with full working codes and proper explanation of each snippet. If we don't find code snippet, we consider them as an article.

Overall, the content and explanation of concepts could have been improved. The content looks too short without enough explanation of concepts.

Hope it's clear.

Thanks.

**Pull request closed without merging**